pkg/nexus/bulk/scheduler/bulk-scheduler.go

```diff
@@ -53,7 +53,7 @@ func (ds *BulkScheduler) executeSchedule() {
 		}
 
```

Comment:
We should take care of the loop, so it not also sleeps, when there are items in the queue, but not enaugh to execute as a bulk

Reply:
I refactored the logic, does the new code looks better now?